app/models/manageiq/providers/amazon/cloud_manager.rb

```diff
@@ -40,6 +40,10 @@ def self.default_blacklisted_event_names
     )
   end
 
+  def self.metrics_collect_queue_name
+    MetricsCollectorWorker.default_queue_name
```

Comment:
As the code is right now it shouldn't get called for those providers, but we will be opening up a method that we know will always fail.